fairroot/eventdisplay/FairEventManager.cxx

```diff
@@ -179,23 +179,23 @@ void FairEventManager::GotoEvent(Int_t event)
     fEntry = event;
     fTimeMin = 0;
     fTimeMax = DBL_MAX;
-    fRunAna->Run(static_cast<Long64_t>(event));
+    fRunAna->RunSingleEntry(static_cast<Long64_t>(event));
```

Comment:
Can you remove the cast now?

```suggestion
    fRunAna->RunSingleEntry(event);
```

Reply:
Or course, in this particular case I've also changed `fRunAna->RunSingleEntry(event);` to `fRunAna->RunSingleEntry(fEntry)'.